security/firely-auth/firely-auth-sso.rst

```diff
@@ -56,12 +73,24 @@ Configuring a new client application in Azure Active Directory (Azure AD) using
     - Select "Token configuration"
     - Select "+ Add optional claim"
     - Select "ID token"
-    - Choose a claim based on which the remote SSO account should be matched against a :ref:`password-less local account <firely_auth_mgmt_sso_user>`. Currently, Firely Auth only supports the "email" claim, but additional  claims may be supported in future. Please make sure to add this claim to the locally provisioned user account.
+    - Choose a claim based on which the remote SSO account should be matched against a :ref:`password-less local account <firely_auth_mgmt_sso_user>`. Currently, Firely Auth supports the ``email`` and ``oid`` claim. Please make sure to add at least one of these claims to the locally provisioned user account.
+    - The ``oid`` claim is populated based on the Object ID that can be found on each user profile in Entra ID. Matching on ``oid`` claim is preferred over the ``email`` claim as it is guaranteed not to change.
+
+#. Configure the ``API permissions`` section
+
+    - Select "API permissions"
+    - Make sure to at least add "email", "profile", and "profile" as permissions
 
 #. Configure the ``ExternalIdentityProviders`` section
 
     - Select "Overview".
     - Select "Endpoints"
     - Use the base url of any of the displayed OAuth 2.0 endpoints as the authority in the settings. It should start with "https://login.microsoftonline.com/" followed by the tenant id within Microsoft Entra ID, ending in "/v2.0/".
 
+#. Optional: Add a `Directory extension <https://learn.microsoft.com/en-us/graph/extensibility-overview?tabs=http#directory-microsoft-entra-id-extensions>`_ for the fhirUser claim owned by the Firely Auth application registered above. 
+
+    The EntraID admin needs to assure that a fhirUser claim is assigned to all accounts that are allowed to be used together with Firely Auth.
```

Comment:
Since this documentation is for admins, I would point out how to do that here, because I don't think it is very straightforward. I would mention that for a particular user they can do a Patch request to

https://graph.microsoft.com/v1.0/users/user object id

with in the body
``
{
    "extension_<extension id of the extension created earlier>_fhirUser": "<fhiruser claim>"
}
``

to set a value to this claim.